Project-I-Social-Card/social-card/src/App.js

```diff
@@ -1,24 +1,26 @@
+// dependency imports
 import React from 'react';
+
+// styling imports
 import './App.css';
 
+// component imports
+import HeaderContainer from './components/HeaderComponents/HeaderContainer';
+import CardContainer from './components/CardComponents/CardContainer';
+import Footer from './components/FooterComponents/Footer';
+
+// compose imported components into one App component
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Social Card!</h3>
-      <p>
-        Begin by exploring the `components` directory. You'll notice we have a
-        few files that we've already included in there to get you started right
-        away building components. You'll need to make sure you include your
-        components that you build in this file to watch your app come to life
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+    <div className="app">
```

Comment:
I love seeing app.js files like these where you have all your logic and components being built in their own files so this is only imported finished components. This is the core of react conceptually and looks great.